src/Tizen.NUI.WindowSystem/src/public/SoftkeyClient.cs

```diff
@@ -0,0 +1,336 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI.WindowSystem.Shell
+{
+    /// <summary>
+    /// Class for the Tizen softkey client.
+    /// </summary>
+    /// This class is need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class SoftkeyClient : IDisposable
+    {
+        private TizenShell _tzsh;
+        private IntPtr _softkeyClient;
+        private int _tzshWin;
+        private bool disposed = false;
+        private bool isDisposeQueued = false;
+
+        /// <summary>
+        /// Visible state of softkey.
+        /// </summary>
+        public enum VisibleState
+        {
+            /// <summary>
+            /// Unknown state. There is no softkey service.
+            /// </summary>
+            /// <since_tizen> 8 </since_tizen>
```

Comment:
Remove since_tizen tags in case of `internal api`

Reply:
I removed all of since_tizen tag in these two classes